File: src/Omnia.ProcessManagement.Models/ProcessTypes/ReviewReminderSchedule/TimeAfterPublishingSchedule.cs
using Omnia.Fx.Models.TimePeriodSettings;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.ProcessTypes.ReviewReminderSchedule
{
    public class TimeAfterPublishingSchedule : ReviewReminderSchedule
    {
        public enum TimeAfterPublishingScheduleTypes
        {

        }

        public override ReviewReminderScheduleTypes Type
        {
            get
            {
                return ReviewReminderScheduleTypes.TimeAfterPublishing;
            }
        }

        public TimePeriodSettings Settings { get; set; }
    }
}

File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processtypes/reviewreminderschedule/TimeAfterPublishingSchedule.ts
import { ReviewReminderSchedule, ReviewReminderScheduleTypes } from './ReviewReminderSchedule';
import { TimePeriodSettings } from '@omnia/fx-models';

export interface TimeAfterPublishingSchedule extends ReviewReminderSchedule {
    type: ReviewReminderScheduleTypes.TimeAfterPublishing
    settings: TimePeriodSettings;
}